src/sentry/options/__init__.py

```diff
@@ -63,4 +63,6 @@
 
 
 def load_defaults():
+    from sentry.hybridcloud import options  # NOQA
```

Comment:
Should this load *after* the defaults are?

Reply:
This ordering is creating from out formatter.  Even if I move it, it is forced back into this order :shrug: